### ISSUE TITLE
Why did it go to Chr0?

### ISSUE THREAD

**Issue by soungalo** [malonge/RaGOO#53]
Hello,
I ran RaGOO v1.1 with default parameters:
`python ragoo.py contigs.fasta ref.fasta -t10`
I looked for two specific contigs in the results which I know should map one after the other to chromosome 4. I looked at the paf output:
```
$ grep -E 'NODE_16239_|NODE_27438_' contigs_against_ref.paf
NODE_16239_length_1784_cov_3.719491     1784    5       1775    -       4       18585056        15903632        15905402        1737      1770    60      tp:A:P  cm:i:177        s1:i:1737       s2:i:0  dv:f:0.0015     rl:i:0
NODE_27438_length_660_cov_4.593388      660     3       645     -       4       18585056        15905406        15906048        642       642     60      tp:A:P  cm:i:67 s1:i:642        s2:i:0  dv:f:0.0008     rl:i:0
```
However, when looking at orderings/4_orderings.txt , only the first contig is there, while the second went to Chr0. Same when I look at groupings/4_contigs.txt.
Can you help me understand why the second contig was added to Chr0? it looks like it maps pretty well to the reference.

Thanks!

**Comment by malonge** [comment]
Hi there,

My guess is that the alignment was too small (< 1 kbp) to be considered. Unfortunately, this is not a tunable parameter (though it will be in v2).

If using default settings, you can manually changes this by editing [line 590](https://github.com/malonge/RaGOO/blob/master/ragoo.py#L590) in `ragoo.py`. Just change `l` to be a smaller number - perhaps 600 in your case. 

Thanks

**Comment by soungalo** [comment]
Thanks for the quick reply. Indeed this worked.
I think this parameter should definitely be externalized.

**Comment by malonge** [comment]
Yeah - definitely an oversight on my part. Hopefully can get v2 released sooner rather than later.
